Fix TODOs for Community Guide (#246)
scientific-python/learn.scientific-python.org: content/community/community-meetings.md

## How to initiate a community meeting?

> - Before starting the meeting, having a few items and discussion points in the

Reviewer: Here we jump straight into an explanation about starting the meeting and an agenda. Should the agenda be distributed beforehand? How? What is the best practice?

> - Before starting the meeting, having a few items and discussion points in the

Reviewer: Do we want to add points about scheduling the meeting, how to plan them (like suggesting alternating timing to allow for different time zones, etc.), and how to best advertise them to the community?

  agenda will help kick-start the meeting. These discussion points could be a
  mix of:
- Issues or PRs from the GitHub repositories which need attention

> - Announcing any upcoming events/conferences to check if someone is going to
>   attend them or is planning to

Reviewer: I think stating the purpose of announcing this will help clarify it here. This is my suggestion but maybe you have a different idea here? 

```suggestion
- Announcing any upcoming events/conferences relevant to the project to check if someone is going to
  attend them or is planning to. People can then connect in person and build community or develop together in person. 
```

- Important updates:
  - These could be recent releases/new feature additions
  - Addition/changes in the maintainer/contributor team

>   - Convergence of an ongoing discussion - a summary of how the community

Reviewer: What do we mean by convergence here? That the update should summarize if convergence exists? Or that the meeting should facilitate a discussion that seeks convergence?

    reached a decision
  - Anything important which needs to be brought up before the community
  - Let everyone know that the meeting is governed by a CoC

## Ice-breakers

- Ice-breakers are usually effective when the participants are not familiar with
  each other
- Apart from the general introduction, which is usually name and role, it’s
  effective to add a fun question (generally non-technical) to ask around;

>   these are:

Reviewer: ```suggestion
  these could include:
```

  - Favourite song, movie, dish, sport, game, tourist spot etc.
  - If you could teach the world one thing, what would it be?
  - What’s one interesting fact about yourself?

## How to address any critical issues in the project/community

> - Sometimes, there are certain matter which needs to be brought up before the
>   community for a more open and inclusive discussion
> - This helps in taking into account various perspectives from the community and
>   choosing a solution which is in the best interest of the community

Reviewer: I don't think these need to be separate points. 

```suggestion
- Sometimes, there are certain matters which need to be brought up directly with the
  community for a more open and inclusive discussion. Bringing these up in a community meeting helps to take into account various perspectives from the community. Ultimately, this may help 
  choose a solution which is in the best interest of the community
```

## How to keep the conversation going?

> - Occasionally, there are situations when the conversation during a community
>   meeting dies, and it feels like you’re in a graveyard 🪦
>   - Having a set up of backup discussion points can help remove the awkward

Reviewer: ```suggestion
- Occasionally, there are situations when the conversation during a community
  meeting dies, and it feels like you’re in a graveyard 🪦. Having a set of backup discussion points can help remove the awkward 
```

    silence
  - Going around in the audience to ask if they have something interesting to
    share with the community
    - What projects are they working on?
    - Something they learned which could be shared with the community
    - Asking around if the attendees need any help

## How to conclude the meeting successfully

- The meeting should end on a positive note - if there has been any heated

>   discussion/arguments, try to conclude before the end. And generally, in the

Reviewer: I think clarifying here would be helpful. Is the point to conclude before arguments end? Or to end on a positive note before the end of the meeting?

>   end, you should:

Reviewer: ```suggestion
  end, one should:
```

  - Conclude important points
  - Revise tabled items - which were supposed to be discussed but didn’t get
    time

>   - Tell them about the next meeting

Reviewer: ```suggestion
  - Announce the next meeting time
  - State what will be covered in the next meeting
```

## How to run office hours?

> - Office hours are a great way to invite the community and answer their

Reviewer: ```suggestion
- Office hours are a great way to invite the community to engage with maintainers and answer their
```

  questions, guide them, help them understand any specific about your OSS
  project etc.
  - Usually, the structure of office hours is open, but there are certain tips
    which could make it run smoother; they are:
  - Greet and acknowledge every participant during the office hours

>   - Try to answer at least 1 question from everyone

Reviewer: ```suggestion
  - Try to answer at least 1 question from each attendee (if time is limited, try to prioritize new attendees)
```
